packaging/test-install.sh

```
@@ -0,0 +1,233 @@
+REPO="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd)"
+
+# The floor the packages declare. Read rather than hardcoded: this file must not be the place
+# the two disagree.
+GLIBC_FLOOR="$(grep -oE 'GLIBC_[0-9]+\.[0-9]+' "$REPO/packaging/nfpm-yorishiro.yaml" | sort -uV | tail -1)"
+
+pass=0 fail=0
+ok()   { printf '  \033[32mPASS\033[0m %s\n' "$1"; pass=$((pass + 1)); }
+bad()  { printf '  \033[31mFAIL\033[0m %s\n' "$1"; fail=$((fail + 1)); }
+note() { printf '\n== %s ==\n' "$1"; }
+
+deb() { ls "$PKG_DIR"/yorishiro_*.deb | head -1; }
```

Shellcheck (0.11.0)

[info] Use find instead of ls to better handle non-alphanumeric filenames.

(SC2012)